src/fifelse.c

```diff
@@ -343,3 +343,161 @@ SEXP fcaseR(SEXP na, SEXP rho, SEXP args) {
   UNPROTECT(nprotect);
   return ans;
 }
+
+SEXP psumR(SEXP na, SEXP args) {
+	if (!IS_TRUE_OR_FALSE(na)) {
```

Comment:
We use 2 whitespace, you used 8. It is so much horizontal scrolling now.

Reply:
I will change all my c files. Thank you for letting me know.